apps/docs/src/cached.ts

```diff
@@ -1,20 +1,15 @@
 /* eslint-disable no-console -- we want to be able to log output to the console */
 import { mkdir, readFile, writeFile } from "fs/promises";
+import * as crypto from "node:crypto";
```

Comment:
```suggestion
import { hash } from "node:crypto";
```

We only need the hash function right?